packages/wallets/src/components/WalletCoolingOffExpirationModal/WalletCoolingOffExpirationModal.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ComponentProps, FC } from 'react';
+import { WalletDialog } from '../Base';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { Button, Text } from '@deriv-com/ui';
+import { FormatUtils } from '@deriv-com/utils';
+import { LabelPairedTriangleExclamationXlFillIcon } from '@deriv/quill-icons';
+import './WalletCoolingOffExpirationModal.scss';
+
+type WalletCoolingOffExpirationModalProps = ComponentProps<typeof WalletDialog> & {
+    coolingOffExpirationDate: number;
+};
+
+const WalletCoolingOffExpirationModal: FC<WalletCoolingOffExpirationModalProps> = ({
+    coolingOffExpirationDate,
+    isVisible,
+    onClose,
+}) => {
+    const { localize } = useTranslations();
+    const expirationDate = FormatUtils.getFormattedDateString(new Date(coolingOffExpirationDate), {
+        format: 'DD MMM YYYY',
+    });
```

Comment:
```suggestion
    const expirationDate = FormatUtils.getFormattedDateString(new Date(coolingOffExpirationDate), {
        format: 'DD MMM YYYY',
    });
```

Need to add method to utils that transforms the date to the format `17 Decemeber` i.e. `DD MMMM`